ansible/template-proxies.yml

```diff
@@ -19,24 +19,22 @@
     DEPLOYED_VERSION: "{{ lookup('env', 'DEPLOYED_VERSION') }}"
     RELEASE_RELEASEID: "{{ lookup('env', 'RELEASE_RELEASEID') }}"
     SOURCE_COMMIT_ID: "{{ lookup('env', 'SOURCE_COMMIT_ID') }}"
-    HOSTED_TARGET_CONNECTION_PATH_SUFFIX: "{{ lookup('env', 'HOSTED_TARGET_CONNECTION_PATH_SUFFIX') }}"
     HOSTED_TARGET_CONNECTION: |
           <SSLInfo>
             <Enabled>true</Enabled>
           </SSLInfo>
           <LoadBalancer>
             <Server name="hosted-apis"/>
           </LoadBalancer>
-          <Path>/{{ service_id }}{{ '-' + pr_number if pr_number else '' }}{{ HOSTED_TARGET_CONNECTION_PATH_SUFFIX }}</Path>
-    HOSTED_TARGET_HEALTHCHECK_SUFFIX: "{{ lookup('env', 'HOSTED_TARGET_HEALTHCHECK_SUFFIX') | default('/_status', True) }}"
+          <Path>/{{ service_id }}{{ '-' + pr_number if pr_number else '' }}{{ HOSTED_TARGET_CONNECTION_PATH_SUFFIX | default('') }}</Path>
     HOSTED_TARGET_HEALTHCHECK: |
       <SSLInfo>
         <Enabled>true</Enabled>
       </SSLInfo>
       <LoadBalancer>
         <Server name="hosted-apis"/>
       </LoadBalancer>
-      <Path>/{{ service_id }}{{ '-' + pr_number if pr_number else '' }}{{ HOSTED_TARGET_HEALTHCHECK_SUFFIX }}</Path>
+      <Path>/{{ service_id }}{{ '-' + pr_number if pr_number else '' }}{{ HOSTED_TARGET_HEALTHCHECK_SUFFIX | default('/_status') }}</Path>
```

Comment:
I think default should be `/healthcheck` which is more common to be present on any app

Reply:
Needs to be left as a default as to not break the apis currently using default. Will flag for a discussion if this should be changed in future though.

Reply:
This is preserving the existing behaviour (see line 31 on the left) -- we can look at changing it in future if more teams are using `healthcheck` than `_status`, but for now I'd prefer to keep the scope of this change small.